### PR TITLE
ArithmeticFunction - createNewRecord

### PR DESCRIPTION
**Pull request by Burdishka**
[Trello US card](https://trello.com/c/sXDMxsq0/711-us026js-entity-arithmetic-function)

This PR contains tests only for creating the new record

### CODE REVIEW

**Review by Lizzzzi — COMMENTED** [review]
imho - do not change other people's files (I mean eventsChain1.js)

and here in this place :
for(let el = 0; el &lt; $(selectors.common.recordActualData).length; el++){            expect($(selectors.common.recordActualData)[el].getText())                .to.equal(expected.arithmeticFunction.expectedValues[el]); 
I think it should be $$ but  not $

**Comment by Burdishka** [comment]
@Lizzzzi сan you please expand your opinion on why we can't make changes to other files?

**Review by SergeiKuznetsov — APPROVED** [review]

**Comment by SergeiKuznetsov** [comment]
> imho - do not change other people's files (I mean eventsChain1.js)

It’s needed for refactoring.

**Review by MariaMoroz — APPROVED** [review]

File: test/specs/entityArithmeticFunction.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        $(selectors.arithmeticFunction.inputField2).click();
+        $(selectors.arithmeticFunction.inputField2)
+            .setValue(testData.arithmeticFunction.f2Value);
+
+        browser.waitUntil(() => $(selectors.arithmeticFunction.div)
+            .getValue() === expected.arithmeticFunction.div);
+
+        $(selectors.common.saveBtn).click();
+
+        expect(getRowCount()).to.equal(1);
```
[the remark this answers is not in the document]
Comment (Burdishka) [reply test/specs/entityArithmeticFunction.js:36]: ok, I will fix it right now

**Review by SamJulia — APPROVED** [review]

**Pull request merged**